lib/msf/core/auxiliary/auth_brute.rb

```diff
@@ -18,7 +18,7 @@ def initialize(info = {})
       OptPath.new('USER_FILE', [ false, "File containing usernames, one per line" ]),
       OptPath.new('PASS_FILE', [ false, "File containing passwords, one per line" ]),
       OptPath.new('USERPASS_FILE',  [ false, "File containing users and passwords separated by space, one pair per line" ]),
-      OptInt.new('BRUTEFORCE_SPEED', [ true, "How fast to bruteforce, from 0 to 5", 5]),
+      OptInt.new('BRUTEFORCE_SPEED', [ true, "How fast to bruteforce, from 0 to 5, anything above 5 is interpreted as minutes", 5]),
```

Comment:
Why don't we just add a new option called something like `BRUTEFORCE_DELAY` and if it is not `nil`, it overrides `BRUTEFORCE_SPEED`?

Reply:
Sounds reasonable. I have another feature that I'm testing out right now which will spray one password for many users and then wait X minutes (as opposed to BRUTEFORCE_SPEED that waits X minutes between each attempt). That way they could be combined to wait X minutes between attempts and then wait Z minutes before going on to the next password in the list. Makes slow spraying efficient.